### PR TITLE
Rynoch's Grave Matter Classes

### PR DESCRIPTION
**Pull request by chankevi**
First Pull

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 2.2.1
+
+- Added Rynoch's Grave Matter's Class's By OE.
```
Comment (mcglincy) [inline CHANGELOG.md:3]: typo: should be "Added Rynoch's Grave Matters classes by OE"

File: module.json [hunk 2]
```diff
@@ -2,76 +2,109 @@
...
   "packs": [
     {
       "name": "mork-borg-3p-actors",
       "label": "MÖRK BORG 3P (Actors)",
+      "path": "packs/mork-borg-3p-actors",
+      "type": "Actor",
       "system": "morkborg",
-      "path": "./packs/mork-borg-3p-actors",
-      "type": "Actor"
+      "ownership": {
```
Comment (mcglincy) [inline module.json:38]: Is this ownership addition actually needed? where did it come from?

**Review by mcglincy — COMMENTED** [review]
Double-checking: is all the new token art public domain and not copyrighted?

Also, "Necromacner.png" is misspelled (should be "Necromancer.png")

File: module.json (hunk 2 above)
Comment (chankevi) [reply module.json:38]: I think this might be a typo error, good catch.

**Comment by chankevi** [comment]
> Double-checking: is all the new token art public domain and not copyrighted?
> 
> Also, "Necromacner.png" is misspelled (should be "Necromancer.png")

All the art is either already within the files or given direct  permission by Rynoch.

**Comment by chankevi** [comment]
This branch has data done outside foundry instead of inside using built in tools. New Branch is properly done and does not break folder structure.

**Pull request closed without merging**